Test of dataverse.harvard.edu is failing

This was first observed when working in a [CI job](https://github.com/jupyterhub/repo2docker/actions/runs/29845120944/job/88683639279?pr=1504) when working on https://github.com/jupyterhub/repo2docker/pull/1504.

### Bug description

```
=========================== short test summary info ============================
FAILED tests/contentproviders/test_dataverse.py::test_fetch[specs0-md5tree0] - AssertionError: assert 'f24541c5f89a...2781220cbbe70' == 'a8f6fc3fc58f...e23fa8b088694'
  
  - a8f6fc3fc58f503cd48e23fa8b088694
  + f24541c5f89a09bc6db2781220cbbe70
============== 1 failed, 13 passed, 1 warning in 72.60s (0:01:12) ==============
```

#### Expected behaviour

Test of dataverse.harvard.edu to pass.

#### Actual behaviour

Test of dataverse.harvard.edu fails.

### How to reproduce

```bash
pytest tests/contentproviders/test_dataverse.py
```

### Your personal set up

- OS: Linux
- Docker version: 29.5.3
- repo2docker version: 0.0.0+unknown

--- a/tests/contentproviders/test_dataverse.py
+++ b/tests/contentproviders/test_dataverse.py
@@ -74,12 +74,13 @@ def test_get_persistent_id(url, persistent_id, is_ambiguous):
     [
         (
             (
-                "doi:10.7910/DVN/TJCLKP",
+                "https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/TJCLKP&version=5.0",
                 "https://dataverse.harvard.edu/citation?persistentId=doi:10.7910/DVN/TJCLKP",
+                "doi:10.7910/DVN/TJCLKP",
             ),
             {
-                "data/primary/primary-data.zip": "a8f6fc3fc58f503cd48e23fa8b088694",
-                "data/2023-01-03.tsv": "6fd497bf13dab9a06fe737ebc22f1917",
+                "data/primary/primary-data.zip": "f24541c5f89a09bc6db2781220cbbe70",
+                "data/2026-06-24.tsv": "0165275ec1437dd2bcb1c6b4b56ef233",
                 "code/language.py": "9d61582bcf497c83bbd1ed0eed3c772e",
             },
         ),

